src/Components/Trainers/index.jsx

```diff
@@ -51,36 +59,52 @@ const Trainers = () => {
     setIsConfirmationModal(true);
   };
 
-  const deleteItem = (id) => {
-    setSelectedTrainerId(id);
-    setModalMessage('Are you sure you want to delete this trainer?');
-    setShowModal(true);
-    setTypeStyle('default');
-    setIsConfirmationModal(true);
-  };
-
   const handleAddTrainer = () => {
-    history.push('/trainers/form');
+    history.push('/src/Components/Trainers/Form/index.jsx');
   };
+
   const editItem = (id) => {
     history.push(`/trainers/form/${id}`);
   };
+
+  const Spinner = () => {
```

Comment:
You can replace this function with the react-spinners library (ClipLoader).
```suggestion
  const Spinner = () => {
```

Reply:
very useful!